src/ServiceStack.Text/PclExport.NetStandard.cs

```diff
@@ -83,6 +84,15 @@ public NetStandardPclExport()
 #else 
             this.DirSep = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? '\\' : '/';
 #endif
+            var req = HttpWebRequest.Create("http://servicestack.net");
+            try
+            {
+                req.Headers[HttpRequestHeader.UserAgent] = "ServiceStack";
```

Comment:
If we did this in the constructor would it prevent the Xamarin Linker from removing it?